src/components/project/project.service.ts

```diff
@@ -565,18 +565,22 @@ export class ProjectService {
       .match([requestingUser(session), ...permissionsOfNode(label)])
       .with('distinct(node) as node')
       .call(projectListFilter, filter)
-      .call(calculateTotalAndPaginateList, input, (q, sort, order) =>
-        q
-          .match([
-            node('node'),
-            relation('out', '', sort, { active: true }),
-            node('prop', 'Property'),
-          ])
-          .with([
-            '*',
-            ...(input.sort === 'sensitivity' ? [sensitivityCase] : []),
-          ])
-          .orderBy(sortBy, order)
+      .call(
+        calculateTotalAndPaginateList,
+        input,
+        this.securedProperties,
+        (q, sort, order) =>
```

Comment:
can't use the defaultSorter on this because it doesn't take in the case statement...yet.  maybe on the next iteration